changed test to quiz: fixes issues in #244

sanjaykdragon opened the pull request:
title, refer to 
https://github.com/rust-lang/rustlings/issues/244

sanjaykdragon commented:
> 
> 
> Wouldn't only renaming the files mess with rustlings watch that is looking for the actual file names to compile?

you are right, this should fix it

fmoko commented:
You would also need to change the reference to the filenames in https://github.com/rust-lang/rustlings/blob/master/README.md#L102.

sanjaykdragon commented:
> 
> 
> You would also need to change the reference to the filenames in [/README.md@`master`#L102](https://github.com/rust-lang/rustlings/blob/master/README.md#L102) [](/rust-lang/rustlings/blob/HEAD@{2020-05-17T10:54:54Z}/README.md#L102).

Didn't notice that, fixed.

> 
> 
> Could you change the title of each exercise? `// test1.rs` to `// quiz1.rs` and so forth?

Forgot about this, done and also changed up the wording from "test about X" to "quiz about X". let me know if I missed anything
